Soccar [Game-Server]/src/nl/soccar/gameserver/model/session/GameWrapper.java

```diff
@@ -70,8 +74,16 @@ public void run() {
     public void stop() {
         timer.cancel();
 
-        game.stop();
         engine.stop();
+
+        playersReady.clear();
+    }
+
+    public void sendWorldObjects(PlayerWrapper player) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)